src/Components/calculator.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+const Calc = () => (
```

Comment:
- [ ] This should be a class component instead of a functional one, kindly make this change 😄